Dockerfile

```diff
@@ -4,7 +4,6 @@ WORKDIR /home/user
 
 # install as a package
 COPY setup.py requirements.txt README.rst /home/user/
-RUN pip install -r requirements.txt
```

Comment:
Let's keep this step in here. It helps to speed up builds when only haystack code changed but requirements didn't (cached step).